packages/rum/test/specs/apm-base.spec.js

```diff
@@ -101,6 +103,10 @@ describe('ApmBase', function () {
       expect(endedTr).toEqual(tr)
       expect(document.readyState).toBe('complete')
       expect(tr.detectFinish).toHaveBeenCalled()
+      expect(window.setTimeout).toHaveBeenCalledWith(
+        jasmine.any(Function),
+        PAGE_LOAD_DELAY
```

Comment:
btw, since this test is not faking setTimeout nor using system.clock the duration of this test will be affected by the real setTimeout. Hence, this test will last an extra second. I think it's not a big deal if the delay remains at that number, but we should be careful with it if in the future we decide to increase the delay for whatever reason.

What do you think? Do you think that we should use the system clock or let this as it is?